astroid/brain/brain_numpy_utils.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2018-2019 hippo91 <guillaume.peillex@gmail.com>
+
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/master/COPYING.LESSER
+
+
+"""Different utilities for the numpy brains"""
+
+
+import astroid
+
+
+def infer_numpy_member(src, node, context=None):
+    node = astroid.extract_node(src)
+    return node.infer(context=context)
+
+
+def looks_like_numpy_member(member_name, node):
+    return (
+        isinstance(node, astroid.Attribute)
+        and node.attrname == member_name
+        and node.expr.inferred()[0].name.startswith("numpy")
```

Comment:
@hippo91 Unfortunately using inference in the transform filters could have certain recursion error side effects, given that `brain` is not fully ready by the time the transform filters are called. Is there any way to prevent having this inference call here?

Reply:
@PCManticore , i changed the way the attribute is meant to be a `numpy` member.
I also introduced docstrings and type annotations.